data/cookbook/extract-links-from-html/00-Re.ml

```diff
@@ -0,0 +1,68 @@
+---
+packages:
+  - name: "re"
+    tested_version: "1.11.0"
+    used_libraries:
+      - re
+---
+
+(*
+
+Given an HTML document or string, we can use the `re` library to create a regular expression that finds the `href` tags containing web links.
+For example, in the sample below we would expect to find three links for this HTML document.
+
+Sample HTML: 
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sample HTML Page</title>
+</head>
+<body>
+    <header>
+        <h1>My Cool Learning Links</h1>
+    </header>
+    <main>
+        <section>
+            <H2>Click a link to get started!</H2>
+            <ul>
+                <li><a href="https://ocaml.org/docs">The Ocaml.org Learning Page</a></li>
+                <li><a href="https://pola.rs/">Pola.rs: Modern Python Dataframes</a></li>
+                <li><a href="https://www.nonexistentwebsite.com">It used to work.com</a></li>
+            </ul>
+        </section>
+    </main>
+</body>
+</html>
+
+`find_links` accepts an argument `html_content` of type string that contains our HTML content.
+
+Using `Re.Perl.re` create a Perl flavored regular expression that searches for the `a href` tags. You can view the pattern using  
+to understand more about what is going on. [Regex101](https://regex101.com/r/2Bs442/1)
```

Comment:
```suggestion
Use `Re.Perl.re` to create a Perl flavored regular expression that searches for the `a href` tags. You can view the pattern using [Regex101](https://regex101.com/r/2Bs442/1)
to understand more about what is going on. 
```

I'm not sure if Regex101 was just in the wrong place or if there is something missing after "You can view the pattern using ... to understand more about what is going on."